packages/app/src/components/ChatPanel.module.css

```
@@ -0,0 +1,326 @@
+  background: #dc2626;
+  color: #fff;
+}
+.cancelBtn:hover {
+  background: #b91c1c;
+}
+
+/* ----------------------------- Dark mode ----------------------------- */
+/* Mirrors DocumentSidebar's dark palette: bg #1e1e1e, border #333,
+   text #ccc, muted #999/#888. */
+
+:global([data-color-scheme="dark"]) .panel {
```

Stylelint (17.10.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)